obstacle_detector/Point2D first_point
obstacle_detector/Point2D last_point

# Parameters of general line equation
float64 A		
float64 B
float64 C
	
float64 variance
int32 sample_size
